miatoll.json

```diff
@@ -0,0 +1,24 @@
+{
+	"response": [
+		{
+			"maintainer": "K O B U X",
+			"oem": "Xiaomi",
+			"device": "Redmi Note 9S/Pro/ProMax / POCO M2 Pro",
+			"filename": "crDroidAndroid-10.0-20200910-miatoll-v6.9.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/miatoll/6.x/crDroidAndroid-10.0-20200910-miatoll-v6.9.zip/download",
+			"timestamp": ,
```

Comment:
timestamp, md5 and size are mandatory tags that need to be completed 
use createjson.sh script to generate those

Reply:
done